test/execution/server_test.exs

```diff
@@ -60,7 +64,10 @@ defmodule Wanda.Execution.ServerTest do
          ]}
       )
 
+      history_log = Repo.all(ExecutionResult)
```

Comment:
you could `assert [] == Repo.all(ExecutionResult)`

Reply:
I tried that but credo was complaining, and it suggested me to use `Enum.empty?` :sweat_smile:

Reply:
oh i see, so you cold `assert ExecutionResult |>  Repo.all() |> Enum.empty?()`

Reply:
oh yeah I like that! Thanks for reminding!

Reply:
addressed in the last commit.